tests/Unit/DataStructures/Test_SpinWeighted.cpp

```diff
@@ -183,7 +183,6 @@ void test_spinweights() {
 
 using SpinWeightedTypePairs =
     tmpl::list<tmpl::list<std::complex<double>, double>,
-               tmpl::list<ComplexDataVector, double>,
```

Comment:
why this change?

Reply:
This comparison no longer succeeds at the overload resolution (it generates a compile-time error). However, this was the only location in the codebase that it was used, so I don't think it's enough of a loss to make the comparison overloads more complicated to address this case.